<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update_s4</class>
 <widget class="QDialog" name="update_s4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_userNo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check UserNo</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>211</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_day">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_year">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_month">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemNo</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_itemNo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_itemNo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Check ItemNo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Review</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_userNo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_new">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>660</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
